### PR TITLE
Workflow Created 

### PR DESCRIPTION
**Pull request by shaggyyy2002**
I have created a workflow as mentioned in the docs Readme.md File #2

### CODE REVIEW

**Comment by saiyam1814** [comment]
Is there anyway we can test this before merging ?

File: .github/workflows/twitter-together.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    name: Tweet
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    steps:
+      - name: checkout master
+        uses: actions/checkout@v2
+      - name: Tweet
+        uses: gr2m/twitter-together@v1.x
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (saiyam1814) [inline .github/workflows/twitter-together.yml]: Did you copy pasted the entire file from the repo ? I would say even if you do please explain all the steps and this secret is not there - it is named TOKEN_GITHUB

Comment (shaggyyy2002) [reply .github/workflows/twitter-together.yml]: The Access Tokens are not visible

Comment (anubha-v-ardhan) [reply .github/workflows/twitter-together.yml]: I believe screenshot of secrets is attached with the issue #2 for reference.

**Comment by shaggyyy2002** [comment]
I think we can check it from actions tab if all the tests are passed or not or we can check by merging if twitter is tweeting about it @saiyam1814

**Review by saiyam1814 — CHANGES_REQUESTED** [review]
Pattern should be same - we create an issue ->  it should create the PR and then on Push that should go out as a Tweet.

File: .github/workflows/twitter-together.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: gr2m/twitter-together@v1.x
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
+  tweet:
+    name: Tweet
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```
Comment (anubha-v-ardhan) [inline .github/workflows/twitter-together.yml]:
```suggestion
    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

File: .github/workflows/twitter-together.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+      - name: checkout master
```
Comment (anubha-v-ardhan) [inline .github/workflows/twitter-together.yml]:
```suggestion
      - name: checkout main
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
We're using `main` as default branch

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!!

@saiyam1814 I believe this PR is ready merge. 
github-actions bot should automatically make a PR adding `tweets/` folder and `tweets/README.md` with the instructions

**Comment by saiyam1814** [comment]
This is on the PR creation? What about when we create issue and that should get converted to PR. People will be creating issues that will be converting to PR and merging it should Tweet out

**Comment by anubha-v-ardhan** [comment]
@saiyam1814 The current setup (if this PR is merged) will work with PRs and not issues. Folks will need to create PRs adding their tweets as `*.tweet` files in `tweet/` folder.
The content will be tweeted on merging that PR.

If we want to configure things to work with issue and use github-action bot to convert it into PRs (like [kubernetes-sigs/contributor-tweets](https://github.com/kubernetes-sigs/contributor-tweets)), we'll need to write those workflows.

**Comment by saiyam1814** [comment]
Yes we need similar workflow , issue creation leads to PR and PR merge leads to Tweet

**Comment by saiyam1814** [comment]
Let's have the PR workflow first - but after this we need to implement the Issue -> PR

**Pull request merged**